csrc/transformer/normalize_kernels.cu

```diff
@@ -92,15 +93,21 @@ __global__ void fused_bias_residual_layer_norm(float* vals,
     if (training)
         if (g.thread_rank() == 0) vars[row] = variance;
 
+    iterations = row_stride / iteration_stride;

```

Comment:
is the re-computation needed?

Reply:
yes, because I changed it in the middle! just wanted to use less registers! :-)